### PR TITLE
Rodauth for team management

### PR DESCRIPTION
**Pull request by sbauch**

### CODE REVIEW

File: client/src/components/TeamManagement/index.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        className={styles.table}
+      >
+        <Table.Column title="Email" dataIndex="email" key="email" />
+        <Table.Column
+          title="Last login"
+          dataIndex="updatedAt"
+          key="updatedAt"
+        />
+        <Table.Column title="Role" dataIndex="role" key="role" />
+        <Table.Column title="Actions" dataIndex="domain" key="domain" />
```
Comment (sccarmichael) [inline client/src/components/TeamManagement/index.tsx:34]: Is it possible to hide "Actions" if the user's not a super admin? If it's blank it feels like a waste of space.

File: client/src/components/TeamManagement/index.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          >
+            <Input id="email" placeholder="teammate@yourco.com" />
+          </Form.Item>
+          <Form.Item
+            label="Role"
+            name="role"
+            rules={[{ required: true, message: 'Choose a role' }]}
+          >
+            <Select>
+              <Select.Option value="base">Base</Select.Option>
```
Comment (sccarmichael) [inline client/src/components/TeamManagement/index.tsx:95]:
```suggestion
              <Select.Option value="base">Default</Select.Option>
```

File: client/src/components/TeamManagement/index.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+      <Table
+        loading={loading}
+        rowKey="id"
+        dataSource={data?.adminUsers}
+        className={styles.table}
+      >
+        <Table.Column title="Email" dataIndex="email" key="email" />
+        <Table.Column
+          title="Last login"
+          dataIndex="updatedAt"
```
Comment (sccarmichael) [inline client/src/components/TeamManagement/index.tsx:30]: IDK how you're running the timestamp here, but is it possible to change to a more human readable format? I.e. "6 hours ago" or even `DD/MM/YY HH:MM`

File: client/src/components/TeamManagement/index.tsx (hunk 1 above)
Comment (sbauch) [reply client/src/components/TeamManagement/index.tsx:34]: its always blank rn, lets just remove it and circle back. we dont yet have any api support for actions (remove, change role?)

also, non admins wont be able to access this screen at all

File: client/src/components/TeamManagement/index.tsx (hunk 3 above)
Comment (sbauch) [reply client/src/components/TeamManagement/index.tsx:30]: these are also faked rn to -12 hours. do you think this column is important to the table?

File: client/src/components/TeamManagement/index.tsx (hunk 1 above)
Comment (sccarmichael) [reply client/src/components/TeamManagement/index.tsx:34]: cool makes sense

File: client/src/components/TeamManagement/index.tsx (hunk 3 above)
Comment (sccarmichael) [reply client/src/components/TeamManagement/index.tsx:30]: not particularly, we can start to add data in as users request it (or during some research I can uncover what is important to include here)

**Review by sccarmichael — APPROVED** [review]
Looks good! #shipit

**Review by sccarmichael — APPROVED** [review]

**Pull request merged**